### PR TITLE
Add component generator

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it adds a simple component generator to stop copying boilerplate.

Usage: run `npm run generate`.

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -187,7 +188,8 @@
   "scripts": {
     "build-assets": "webpack -p",
     "build-schema": "python manage.py get_graphql_schema > saleor/graphql/schema.graphql",
-    "generate-types": "apollo client:codegen --target=typescript types --globalTypesFile=saleor/static/dashboard-next/types/globalTypes.ts",
+    "build-types": "apollo client:codegen --target=typescript types --globalTypesFile=saleor/static/dashboard-next/types/globalTypes.ts",
+    "generate": "plop --plopfile .plop/plopfile.js",
```
Comment (maarcingebala) [inline package.json]: I think `npm run generate` sounds too enigmatic. I'd propose using a more explicit command like `generate-component` for that.

Comment (dominik-zeglen) [reply package.json]: At this moment it generates only components, but in the future, it can be used to generate other things.

Comment (maarcingebala) [reply package.json]: So how will we tell the generator what files to generate then? Still, I think `generate` doesn't give any guesses about what it actually does.

Comment (maarcingebala) [reply package.json]: Another idea: `npm run new-component`

Comment (piotrgrundas) [reply package.json]: I agree with @maarcingebala with renaming it sth like `generate-component`.
OR u could leave generate, but change the prompt in such way, that ATM u could only choose creation of a new component.

**Pull request merged**